.travis.yml

```diff
@@ -2,7 +2,7 @@ language: scala
 scala:
   - 2.11.6
 script:
-  - sbt ++$TRAVIS_SCALA_VERSION validate publishLocal
+  - sbt ++${TRAVIS_SCALA_VERSION} coverage validate coverageReport coverageAggregate && bash <(curl -s https://codecov.io/bash) && sbt clean validate publishLocal ;
```

Comment:
> why add brackets here?

No particular reason, other than some bits that I did have and later removed, also used that style. I'll revert to no-bracket

> Has sbt-scoverage found a way to support coverageAggregate

I read that too but it seems to work fine - will have a closer inspection later